### PR TITLE
Change in guacaomole.md; fixes #13

### LINKED ISSUE #13: Add more detais to the guacamole ingredients.
Specify vegetable type and freshness - for easier grocery shopping.

### PR DESCRIPTION
**Pull request by Dreytte**

### CODE REVIEW

File: sides/guacamole.md [hunk 1]
```diff
@@ -3,12 +3,12 @@
 
 ## Ingredients
 
-- 2 avocados
-- 1 chili
+- 2 avocados (hass)
+- 1 chili (jalapenos)
```
Comment (cesarmarin23) [inline sides/guacamole.md:7]: I'm allergic so we should delete it or change it to another spicy ingredient

**Review by emmaaer — APPROVED** [review]
Looks delicious!

File: sides/guacamole.md (hunk 1 above)
Comment (Dreytte) [reply sides/guacamole.md:7]: What a shame! Are you allergic to avocado or chili?

Comment (cesarmarin23) [reply sides/guacamole.md:7]: chili :(

Comment (Dreytte) [reply sides/guacamole.md:7]: what about paprika?

**Pull request merged**